### ISSUE TITLE
Why am I not receiving correctly 'ñ' character?

### ISSUE THREAD

**Issue by MackYoel** [falconry/falcon#653]
I just want to use the URL parameter, but the 'ñ' character is received like 'Ã±', how can I receive correctly 'ñ'? 
An use case for example: /users/ñoño GET.
The code is something like this.
<pre>
class Item:
    def on_get(self, req, resp, username):
        print(username) # Ã±oÃ±o
        print(username == 'Ã±oÃ±o') # True
</pre>

**Comment by philiptzou** [comment]
I think this one is fixed in #613. What version of Falcon are you using?

**Comment by MackYoel** [comment]
I am using falcon 0.3.0 and python 3.4.0

**Comment by philiptzou** [comment]
Well I think this fix is not included in 0.3.0. So you can either install the latest version or patch your Falcon.

To install the latest version:
```bash
pip install git+https://github.com/falconry/falcon.git
```

To patch your Falcon with pep3333 fix:
```bash
wget https://github.com/falconry/falcon/commit/6b011038f691d03174da26f06ddf45528d3cefd3.diff -O /tmp/patch-pep3333.diff
cd /path/to/site-packages/falcon
patch -p1 < /tmp/patch-pep3333.diff
```

**Comment by MackYoel** [comment]
It worked! thank you very much @philiptzou!!!
